src/routes/UserRoutes.ts

```diff
@@ -32,6 +32,12 @@ export class UserRoutes {
 
 		router.get('/users/:id', getUser(TokenType.Auth), isVerified, this.userController.getUser.bind(this.userController));
 
+		router.get('/users/@me/notes', getUser(TokenType.Auth), isVerified, this.userController.getNotes.bind(this.userController));
+
+		router.post('/users/:id/note', getUser(TokenType.Auth), isVerified, this.userController.createNote.bind(this.userController));
```

Comment:
```suggestion
		router.put('/users/@me/notes/:id', getUser(TokenType.Auth), isVerified, this.userController.createNote.bind(this.userController));
```

This is more of a structural change. If we want to create a note for ourselves, it should come under @me. Also, since we are specifying the exact location of the new resource, it is technically a PUT, not a POST.